lib/server.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra/base'
+
+class ChitterChallenge < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
+    erb :index
+  end
+
+  get '/sign-up' do
+    erb :'sign-up'
+  end
+
+  post '/sign-up' do
+    session[:username] = params[:username]
+    redirect to '/signed-in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.